core/src/main/java/org/mule/transformer/simple/AddSessionVariableTransformer.java

```diff
@@ -6,20 +6,29 @@
  */
 package org.mule.transformer.simple;
 
-import org.mule.PropertyScope;
+import org.mule.api.MuleEvent;
+import org.mule.api.metadata.DataType;
+
+import java.io.Serializable;
 
 public class AddSessionVariableTransformer extends AbstractAddVariablePropertyTransformer
 {
 
+    public void setVariableName(String variableName)
```

Comment:
Is strange in general to have methods talking about "variable" and others talking about "property"